repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/ClientAuthenticationHandlerFactory.java

```diff
@@ -77,7 +78,12 @@ public void configurationUpdated(ClientAuthConfig modifiedConfig) {
                     accountRegexExtractor.addPattern(clientMapping.getIdRegex());
                 }
                 if (modifiedConfig.getAtomFeeds() != null) {
-                    activateOpenstackAtomFeedListener(modifiedConfig);
+                    try {
+                        activateOpenstackAtomFeedListener(modifiedConfig);
+                    } catch (TimeoutException e) {
```

Comment:
This was originally just bubbling up the NPE as an unchecked `Exception`. If we don't consume it here, then we have to add it to the `UpdateListener` interface. It just seems kind of strange to burden all `UpdateListener`'s for the behavior of this one implementor. Should we re-throw an unchecked exception here? That seems kind of odd too.